### ISSUE TITLE
[a11y] Query results missing link

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1463]
![image](https://user-images.githubusercontent.com/37256050/167482758-0a860a5b-c161-4c7a-bd90-b2ea76ea42b4.png)

The `a11y` instance is missing the linkout button on the left of each query result.

db: nationalshell
username: nationalshell

**Comment by maxxxxxdlp** [comment]
This is because you run the query in a distinct mode
Since record IDs are unique, when you are looking for distinct results, record IDs are not returned, and thus you can't link out to them

**Comment by grantfitzsimmons** [comment]
The `edge `still can link out from the returned records

**Comment by maxxxxxdlp** [comment]
Distinct queries are broken on edge (that checkbox has no effect on edge)
See: https://github.com/specify/specify7/issues/331
